[32mINFO    [0m juju.model:__init__.py:3254 Waiting for model:
  self-signed-certificates/0 [idle] active: 
[32mINFO    [0m juju.model:__init__.py:3254 Waiting for model:
  vault-k8s/0 [idle] active: 
  vault-k8s/1 [idle] active: 
  vault-k8s/2 [idle] active: 
[32mINFO    [0m tests.integration.vault:vault.py:94 Unsealed vault unit: https://10.1.249.16:8200.
[32mINFO    [0m tests.integration.vault:vault.py:76 Vault unit is unsealed.
[32mINFO    [0m tests.integration.vault:vault.py:76 Vault unit is unsealed.
[32mINFO    [0m tests.integration.vault:vault.py:94 Unsealed vault unit: https://10.1.249.25:8200.
[32mINFO    [0m tests.integration.vault:vault.py:76 Vault unit is unsealed.
[32mINFO    [0m tests.integration.vault:vault.py:94 Unsealed vault unit: https://10.1.249.23:8200.
[32mINFO    [0m tests.integration.vault:vault.py:76 Vault unit is unsealed.
[32mINFO    [0m juju.model:__init__.py:3254 Waiting for model:
  vault-k8s/0 [idle] blocked: Please unseal Vault
  vault-k8s/1 [idle] blocked: Please unseal Vault
  vault-k8s/2 [idle] blocked: Please unseal Vault
[32mINFO    [0m juju.model:__init__.py:3254 Waiting for model:
  vault-k8s/0 [idle] active: 
  vault-k8s/1 [idle] active: 
  vault-k8s/2 [idle] active:
[32mINFO    [0m pytest_operator.plugin:plugin.py:937 Model status:

Model           Controller                Cloud/Region        Version  SLA          Timestamp
test-core-6fk6  github-pr-b2479-microk8s  microk8s/localhost  3.6.6    unsupported  17:54:37Z

App                       Version  Status  Scale  Charm                     Channel   Rev  Address         Exposed  Message
self-signed-certificates           active      1  self-signed-certificates  1/stable  263  10.152.183.232  no       
vault-k8s                          active      3  vault-k8s                             0  10.152.183.246  no       

Unit                         Workload  Agent  Address      Ports  Message
self-signed-certificates/0*  active    idle   10.1.249.31         
vault-k8s/0*                 active    idle   10.1.249.16         
vault-k8s/1                  active    idle   10.1.249.25         
vault-k8s/2                  active    idle   10.1.249.23         

[32mINFO    [0m pytest_operator.plugin:plugin.py:943 Juju error logs:

unit-vault-k8s-1: 17:47:48 ERROR unit.vault-k8s/1.juju-log [vault_client] Error while checking Vault health status: HTTPSConnectionPool(host='vault-k8s-1.vault-k8s-endpoints.test-core-6fk6.svc.cluster.local', port=8200): Max retries exceeded with url: /v1/sys/health (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3104ed6d50>: Failed to establish a new connection: [Errno 111] Connection refused'))
unit-vault-k8s-2: 17:48:36 ERROR juju.worker.dependency "log-sender" manifold worker returned unexpected error: sending log message: websocket: close 1005 (no status): websocket: close sent
model-95f07c02-eeb0-406c-8500-b0e4976d9984: 17:48:36 ERROR juju.worker.dependency "log-sender" manifold worker returned unexpected error: sending log message: websocket: close 1005 (no status): websocket: close sent
unit-vault-k8s-0: 17:48:38 ERROR juju.worker.dependency "log-sender" manifold worker returned unexpected error: sending log message: websocket: close 1005 (no status): websocket: close sent
unit-self-signed-certificates-0: 17:51:17 ERROR unit.self-signed-certificates/0.juju-log At least 1 matching relation ID not found with the relation name 'send-ca-cert'
unit-vault-k8s-2: 17:51:23 ERROR juju.worker.dependency "log-sender" manifold worker returned unexpected error: sending log message: websocket: close 1005 (no status): websocket: close sent
unit-vault-k8s-1: 17:51:23 ERROR juju.worker.dependency "log-sender" manifold worker returned unexpected error: sending log message: write tcp 10.1.249.25:39860->10.152.183.137:17070: write: connection reset by peer: write tcp 10.1.249.25:39860->10.152.183.137:17070: write: connection reset by peer
model-95f07c02-eeb0-406c-8500-b0e4976d9984: 17:51:24 ERROR juju.worker.dependency "log-sender" manifold worker returned unexpected error: sending log message: websocket: close 1005 (no status): websocket: close sent
unit-vault-k8s-0: 17:51:24 ERROR juju.worker.dependency "log-sender" manifold worker returned unexpected error: sending log message: websocket: close 1005 (no status): websocket: close sent

[32mINFO    [0m pytest_operator.plugin:plugin.py:1049 Resetting model test-core-6fk6...
[32mINFO    [0m pytest_operator.plugin:plugin.py:1038    Destroying applications vault-k8s
[32mINFO    [0m pytest_operator.plugin:plugin.py:1038    Destroying applications self-signed-certificates
[32mINFO    [0m pytest_operator.plugin:plugin.py:1054 Not waiting on reset to complete.
[32mINFO    [0m pytest_operator.plugin:plugin.py:1025 Forgetting model main...